### PR TITLE
Day in Life of Release Manager blog and images

### LINKED ISSUE #740: Release Manager Blog Post
**Describe the blog post you would like to write**
This blog post will be about being release manager for the first time in OpenSearch and the mistakes I made and the things I learned.

**What is the title of the blog post?**
Day in the Life of a Release Manager

**Who are the authors?**
@eugenesk24
@stockholmux 

**What is the proposed posting date?**
As soon as possible.

### PR DESCRIPTION
**Pull request by eugenesk24**
Signed-off-by: Eugene Lee <eugenesk@amazon.com>

### Description
New blog post describing experience as first time release manager
 
### Issues Resolved
#740 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2022-08-25-day-in-the-life-of-a-release-manager.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+After forking the [repository](https://github.com/opensearch-project/opensearch-build), I check out a branch, add my team’s plugins to the manifest, and create a [pull request](https://github.com/opensearch-project/opensearch-build/issues/889#issuecomment-1036510660) (PR).
+
+Two weeks before release, [GitHub issues](https://github.com/opensearch-project/observability/issues/503) are created in each repository with a checklist for each (the actual checklist that was created for this version release is pictured below). I self-assign the issue and check off some of the completed tasks. In hindsight, it seems obvious that I would know that checking off a task does not mean that it is actually completed. But this only becomes clear to me when I wake up one morning to the continuous integration (CI) builds failing.
+
+![Image: Figure 1]({{ site.baseurl }}/assets/media/blog-images/2022-08-24-day-in-the-life-of-a-release-manager/release-checklist-1.png){: .img-fluid }
```
Comment (krisfreedain) [inline _posts/2022-08-25-day-in-the-life-of-a-release-manager.md:27]: "2022-08-24" - looks like this should be 2022-08-25 on each of your images

**Review by krisfreedain — CHANGES_REQUESTED** [review]
Small change to the path on each image file

File: _posts/2022-08-25-day-in-the-life-of-a-release-manager.md (hunk 1 above)
Comment (eugenesk24) [reply _posts/2022-08-25-day-in-the-life-of-a-release-manager.md:27]: Good catch, thank you!

**Review by krisfreedain — APPROVED** [review]
looks great!

**Pull request merged**